src/opnsense/mvc/app/models/OPNsense/Core/ACL/ACL.xml

```diff
@@ -632,30 +632,32 @@
             <pattern>api/diagnostics/log/core/pkg/*</pattern>
         </patterns>
     </page-system-firmware-manualupdate>
+    <page-system-gateways>
```

Comment:
up for discussion how useful it is to add two acl's. I think we dropped one in the other refactors.